app/serializers/bike_v2_serializer.rb

```diff
@@ -58,13 +58,11 @@ def date_stolen_string
   end
 
   def placeholder_image
-    svg_path =
-      Rails
-        .application
-        .assets["revised/bike_photo_placeholder.svg"]
-        .digest_path
+    assets = Rails.application.assets
+    return if assets.blank?
 
-    "#{ENV["BASE_URL"]}/assets/#{svg_path}"
+    svg_path = assets["revised/bike_photo_placeholder.svg"]&.digest_path
```

Comment:
Should we create some sort of helper for this? Or some sort of caching? Does this create any performance problems?

Reply:
Nevermind, this is resolved by #1223

Reply:
This was an expedient fix — https://github.com/bikeindex/bike_index/pull/1223 takes a better approach.